kahuna/app/views/main.scala.html

```diff
@@ -15,8 +15,8 @@
     <link rel="preconnect" href="@mediaApiUri"/>
     <link rel="preconnect" href="@authApiUri"/>
 
-    <link rel="shortcut icon" type="image/png" href="@routes.Assets.at("images/grid-favicon-32.png")"/>
-    <link rel="assets" href="@routes.Assets.at("")"/>
+    <link rel="shortcut icon" type="image/png" href="@routes.Assets.versioned("images/grid-favicon-32.png")"/>
```

Comment:
I thiiiiink the way asset fingerprinting works is that we'll have multiple versions only when the favicon changes.